google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseClientImpl.java

```diff
@@ -112,6 +126,16 @@ public Timestamp writeAtLeastOnce(final Iterable<Mutation> mutations) throws Spa
   public CommitResponse writeAtLeastOnceWithOptions(
       final Iterable<Mutation> mutations, final TransactionOption... options)
       throws SpannerException {
+    if (useMultiplexedSessionForBlindWrite) {
+      return getMultiplexedSession().writeAtLeastOnceWithOptions(mutations, options);
+    } else {
+      return writeAtLeastOnceWithSession(mutations, options);
+    }
+  }
+
+  public CommitResponse writeAtLeastOnceWithSession(
```

Comment:
Can this be private? Or at least package-private?

Reply:
Done.